racket/collects/racket/contract/private/provide.rkt

```diff
@@ -291,20 +300,20 @@
                                                 stx id reflect-id ctrct/no-prop user-rename-id
                                                 pos-module-source
                                                 mangle-for-maker?
-                                                provide?)
+                                                upe-id)
   (define ex-id (or reflect-id id))
   (define id-rename-without-source (id-for-one-id user-rename-id reflect-id id mangle-for-maker?))
   (define id-rename (datum->syntax id-rename-without-source
                                    (syntax-e id-rename-without-source)
                                    ex-id
                                    id-rename-without-source))
+  (define ext-name (or user-rename-id id))
   (with-syntax ([ctrct (syntax-property 
                         (syntax-property
                          ctrct/no-prop
                          'racket/contract:contract-on-boundary
                          (gensym 'provide/contract-boundary))
-                        'inferred-name ex-id)]
-                [external-name (or user-rename-id id)])
+                        'inferred-name ex-id)])
```

Comment:
Why did you remove `external-name` in favor of `ext-name`? I also see three arrows comes out of that binding in DrRacket but only two of them are changed in the below.